omod/src/main/resources/config.xml

```diff
@@ -146,5 +146,16 @@
         <defaultValue></defaultValue>
     </globalProperty>
 
+    <globalProperty>
+        <property>ugandaemr.dsdm.programsuuid</property>
+        <description>List of Programs in DSDM</description>
```

Comment:
@slubwama Suggested change - comma delimited list of program uuids for DSDM (are these program or concept uuids?)